osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -102,15 +126,27 @@ public abstract class Flashlight : Drawable
 
             public List<BreakPeriod> Breaks = new List<BreakPeriod>();
 
-            private readonly float defaultFlashlightSize;
```

Comment:
why is the flashlight part of the name removed, a nitpick for sure but id leave it in for clarity

Reply:
Fixed.